azurerm/internal/tf/validation/pluginsdk.go

```diff
@@ -0,0 +1,208 @@
+package validation
```

Comment:
Wondering why this file is not located in the `pluginsdk` folder?

Reply:
so `validation` made sense as a separate package to me since this is what I think you'd expect to search for? arguably we could put this `validation` package within the `pluginsdk` folder, but whilst there's some benefits to having the wrapper in a single package - ultimately I don't think that'd be as discoverable tbh?

Reply:
I agree that it make sense to put these code to a package with some reasonable name, which helps it easier to find. Whilst it makes it a bit harder to tear down the wrappers later, once we've got migrated all the code to v2. This is some kind of a tradeoff.

Reply:
FWIW there's some larger questions we're trying to answer regarding the Typed vs Untyped resources - ultimately we're intending on migrating over to Typed resources at some point (so migrating off the wrapper becomes less useful in favour of ditching it in favour of the Typed SDK), but there's some upstream questions to resolve first (which is why this is in limbo at the moment)